src/main/java/org/jenkinsci/plugins/matrixauth/integrations/casc/MatrixAuthorizationStrategyConfigurator.java

```diff
@@ -0,0 +1,60 @@
+package org.jenkinsci.plugins.matrixauth.integrations.casc;
+
+import hudson.security.AuthorizationStrategy;
+import hudson.security.Permission;
+import io.jenkins.plugins.casc.Attribute;
+import io.jenkins.plugins.casc.BaseConfigurator;
+import io.jenkins.plugins.casc.impl.attributes.MultivaluedAttribute;
+import org.jenkinsci.plugins.matrixauth.AuthorizationContainer;
+import org.jenkinsci.plugins.matrixauth.integrations.PermissionFinder;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import javax.annotation.Nonnull;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Restricted(NoExternalUse.class)
+public abstract class MatrixAuthorizationStrategyConfigurator<T extends AuthorizationContainer> extends BaseConfigurator<T> {
+
+    @Nonnull
+    @Override
+    public Class<?> getImplementedAPI() {
+        return AuthorizationStrategy.class;
+    }
+
+
+    @Override
+    @Nonnull
+    public Set<Attribute<T, ?>> describe() {
+        return Collections.singleton(
+                new MultivaluedAttribute<T, String>("grantedPermissions", String.class)
```

Comment:
```suggestion
                new MultivaluedAttribute<T, String>("permissions", String.class)
```

Already done for the `NodeProperty`, just not here.

Reply:
this will break those that currently use it :cry: but you are of course allowed, just make it clear :)